course/src/main/java/in/testpress/course/ui/ContentActivity.java

```diff
@@ -346,51 +354,80 @@ protected boolean shouldOverrideUrlLoading(Activity activity, String url) {
         }
     }
 
+    private void setChapterNameInActionBarTitle(long chapterId) {
+        List<Chapter> chapters = TestpressSDKDatabase.getChapterDao(this).queryBuilder()
+                .where(ChapterDao.Properties.Id.eq(chapterId)).list();
+
+        if (!chapters.isEmpty()) {
+            //noinspection ConstantConditions
+            getSupportActionBar().setTitle(chapters.get(0).getName());
+        }
+    }
+
     private void checkContentType() {
+        setChapterNameInActionBarTitle(content.getChapterId());
+        setContentTitle(content.getTitle());
```

Comment:
Should be setTitle

Reply:
setTitle is android's default method to change activity's title